projects/packages/videopress/src/client/admin/components/video-details/types.ts

```diff
@@ -9,6 +9,11 @@ export type VideoDetailsProps = {
 	 */
 	src: string;
 
+	/**
+	 * VideoPress embed shortcode.
+	 */
+	shortcode: string;
```

Comment:
Based on `edit-video-details` it could be null, so it should be optional, not?

Reply:
Now it cannot be `null` anymore.